### PR TITLE
Add select multiple

### PR DESCRIPTION
**Pull request by edetec**

### CODE REVIEW

File: src/ng-xform/fields/select-field.ts [hunk 1]
```diff
@@ -1,14 +1,30 @@
...
-  constructor(options = {}) {
+  constructor(options: SelectField = {}) {
     super(options);
-    this.options = options['options'];
+    this.options = options.options;
+    this.multiple = options.multiple || false;
+    this.noFilterUntil = options.noFilterUntil || 0;
+    this.valueAttribute = options.valueAttribute;
+    this.labelAttribute = options.labelAttribute;
+    this.tagLabelAttribute = options.tagLabelAttribute;
```
Comment (igortg) [inline src/ng-xform/fields/select-field.ts]: Do we really need `labelAttribute` and `tagLabelAttribute`?

File: src/ng-xform/ng-xform.component.ts [hunk 2]
```diff
@@ -72,7 +72,7 @@ export class NgXformComponent implements OnInit, OnChanges {
   private getAttributeValue(attr: string, value: any): any {
```
Comment (igortg) [inline src/ng-xform/ng-xform.component.ts:72]: Didn't you removed this method? It is in another branch?

Comment (edetec) [reply src/ng-xform/ng-xform.component.ts:72]: yep

File: src/ng-xform/fields/select-field.ts (hunk 1 above)
Comment (edetec) [reply src/ng-xform/fields/select-field.ts]: No, I'll remove it.

**Review by lvoliveira — APPROVED** [review]

**Review by igortg — APPROVED** [review]

**Pull request merged**